addons/survey/views/survey_templates_statistics.xml

```diff
@@ -99,46 +86,82 @@
+                            <span t-out="len(question_data['answer_line_ids'].mapped('user_input_id'))"/>
+                            <span class="text-muted">Responded</span>
+                        </span>
+                    </div>
+                    <!-- Numerical KPIs -->
+                    <span t-if="question.question_type in ['numerical_box', 'scale']" class="d-flex gap-1 my-1 me-2">
+                        <div class="btn-group o_survey_results_question_pill" role="group" aria-label="Maximum">
+                            <span class="badge text-bg-secondary only_left_radius px-2 pt-1">Max </span>
+                            <span class="badge text-bg-success only_right_radius px-2 pt-1" t-esc="question_data['numerical_max']"></span>
+                        </div>
+                        <div class="btn-group o_survey_results_question_pill" role="group" aria-label="Minimum">
+                            <span class="badge text-bg-secondary only_left_radius px-2 pt-1">Min </span>
+                            <span class="badge text-bg-danger only_right_radius px-2 pt-1" t-esc="question_data['numerical_min']"></span>
+                        </div>
+                        <div class="btn-group o_survey_results_question_pill" role="group" aria-label="Average">
+                            <span class="badge text-bg-secondary only_left_radius px-2 pt-1">Avg </span>
+                            <span class="badge text-bg-warning only_right_radius px-2 pt-1" t-esc="question_data['numerical_average']"></span>
+                        </div>
                     </span>
+                    <!-- Data buttons -->
+                    <ul t-if="question.question_type not in ['text_box', 'char_box'] and question_answered"
+                        class="nav btn-group flex-nowrap d-print-none ms-auto" role="tablist">
+                        <li t-if="question.question_type in ['simple_choice', 'multiple_choice', 'matrix', 'scale']" class="nav-item btn-group">
+                            <a t-att-href="'#survey_graph_question_%d' % question.id" t-att-aria-controls="'survey_graph_question_%d' % question.id"
+                                data-bs-toggle="tab" role="tab" class="nav-link btn btn-light py-1 px-2 active">
+                                <i t-if="question.question_type == 'simple_choice'" class="fa fa-pie-chart"/>
+                                <i t-else="" class="fa fa-bar-chart-o"/>
+                            </a>
+                        </li>
+                        <li t-elif="question.question_type in ['numerical_box', 'date', 'datetime']" class="nav-item btn-group">
+                            <a t-att-href="'#survey_stats_question_%d' % question.id" t-att-aria-controls="'survey_stats_question_%d' % question.id"
+                                data-bs-toggle="tab" role="tab" class="nav-link btn btn-light py-1 px-2 active">
+                                <i class="fa fa-trophy"/>
```

Comment:
I'm not sold on that trophy, is it really obvious that this table shows the 5 most common answers? Maybe we could at least keep a title on mouse hover "Most Common" here?

Or we could change the label in the table to "Top 5 User Responses" here?
![image](https://github.com/odoo/odoo/assets/41282536/26e1fa41-50a5-4a95-b5fe-2824b7dde4b7)

(Maybe something to double-check with François).